emloop/tests/cli/common_test.py

```diff
@@ -343,3 +355,20 @@ def test_config_file_is_incorrect(tmpdir, caplog):
     config['main_loop']['on_incorrect_config'] = 'ignore'
     run(config=config, output_root=tmpdir)
     assert 'Extra arguments: {\'non-existent\': \'none\'}' not in caplog.text
+
+
+def test_run_with_eval_stream(tmpdir, caplog):
+    """Test that eval is called with given stream and not with EL_DEFAULT_TRAIN_STREAM."""
+    caplog.set_level(logging.INFO)
+
+    config = {'dataset': {'class': 'emloop.tests.cli.common_test.DummyEvalDataset'},
+              'stream': {'train': {'rotate': 20}},
```

Comment:
`stream` is kind of unusual config item, what is it supposed to mean? Did you mean `eval` ?